packages/overlay/README.md

```diff
@@ -95,6 +95,10 @@ type OverlayOptions = {
 
 `receivesFocus` tells the overlay stack to throw focus into the overlay after it has opened.
 
+### Events
+
+The work to both open and close an overlay is asyncronous. This asyncrony is surfaced into the application via DOM events dispatched from the `trigger` element of your overlay. An `sp-opened` event will be dispatched once the overlay has completed all of the work involved in opening an overlay, and an `sp-closed` event will be dispatched onced the overlay has finished closing. In both cases the dispatched event will include a `detail` property with an `interaction: TriggerInteractions` key to support associating the event/overlay with its originating `interaction`.
```

Comment:
asynchronous/asynchrony are misspelled. 

I would also maybe reword " An `sp-opened` event will be dispatched once the overlay has completed all of the work involved in opening an overlay" to "An `sp-opened` event will be dispatched once the overlay has done the work to open an overlay" to make it a little less wordy.

Reply:
Cool. Took the reduction a little further in that sentence, but great instinct.